bin/ch-pull2tar

```diff
@@ -20,12 +20,46 @@ EOF
 
 parse_basic_args "$@"
 
-if [ "$#" -ne 2 ]; then
+if [ "$#" -lt 2 ]; then
     usage
 fi
 
 image=$1
 dest=$2
+tar="$image".tar.gz
 
-docker_ pull "$image"
-"${ch_bin}/ch-builder2tar" "$image" "$dest"
+while [ $# -gt 0 ]; do
+    opt=$1; shift
+    case $opt in 
+        -p|--pull-from) 
+            if [ "$1" = "docker" ]; then
+                docker=yes
```

Comment:
I'd have a single variable to contain the puller (i.e., values `docker` or `skopeo`). Eventually we are going to have ` `CH_PULLER` environment variable or similar, so prepare for that.

Reply:
Should I put in the plumbing for that right now, or should we just defer that to later?

Reply:
I'd say yes. I think it will make your work on this PR easier.

Reply:
Will do!